### PR TITLE
Escaping package names for windows packages

### PR DESCRIPTION
**Pull request by frezbo**
Signed-off-by: Noel Georgi <18496730+frezbo@users.noreply.github.com>

Fixes: https://github.com/inspec/inspec/issues/3258

### CODE REVIEW

**Comment by frezbo** [comment]
@jquick @jerryaldrichiii Do you want me to update the tests, please take a look at the issue linked here.

**Comment by frezbo** [comment]
This should be good to go.

File: lib/resources/package.rb [hunk 1]
```diff
@@ -288,7 +288,7 @@ def info(package_name)
       # Find the package
       cmd = inspec.command <<-EOF.gsub(/^\s*/, '')
         Get-ItemProperty (@("#{search_paths.join('", "')}") | Where-Object { Test-Path $_ }) |
-        Where-Object { $_.DisplayName -match "^\\s*#{package_name}\\s*$" -or $_.PSChildName -match "^\\s*#{package_name}\\s*$" } |
+	Where-Object { $_.DisplayName -match "^\\s*#{package_name.shellescape}\\s*$" -or $_.PSChildName -match "^\\s*#{package_name.shellescape}\\s*$" } |
```
Comment (jerryaldrichiii) [inline lib/resources/package.rb]: Slight nitpick, but if you could indent this line to match the others I would be most delighted 😄

Comment (frezbo) [reply lib/resources/package.rb]: @jerryaldrichiii , argg!! seems a mistake from my side, fixed. :+1:

**Review by jerryaldrichiii — APPROVED** [review]
Looks good to me, thanks @frezbo!

**Review by jquick — APPROVED** [review]
Thanks @frezbo

**Pull request merged**